requirements.txt

```diff
@@ -0,0 +1,4 @@
+asgiref==3.4.1
+Django==4.0.1
+sqlparse==0.4.2
+tzdata==2021.5
```

Comment:
Should pylance and the other extensions of vscode be listed here, excuse if my question is stupid

Reply:
This mustn't be in the requerements.txt file, if u want that can be placed in the README.md to help developers work on our code, but if they use vscode they might have installed them already.

Reply:
You are right, actually vscode suggests installing those. No need to add them, You convinced me 100%